### ISSUE TITLE
In-Game Credits

### ISSUE THREAD

**Issue by vbawol** [EpochModTeam/DayZ_SA_Epoch#13]
Develop a way to show credits for the mod development in-game.

**Comment by GrosTon1** [comment]
Can we discuss the way to do so ? A button in the main screen, a keybind that works even if in game, a button and/or a text available in the escape menu, a combination of some things ? That should be really easy to do even with the current state of the game.

**Comment by vbawol** [comment]
Certainly, I think a button on the main menu and the escape menu would be good enough.
